app/main.py

```diff
@@ -1,5 +1,63 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords or [0, 0]
+
+    def go_forward(self, coord_y: int = 1) -> None:
+        self.coords[1] += coord_y
+
+    def go_back(self, coord_y: int = 1) -> None:
+        self.coords[1] -= coord_y
+
+    def go_right(self, coord_x: int = 1) -> None:
+        self.coords[0] += coord_x
+
+    def go_left(self, coord_x: int = 1) -> None:
+        self.coords[0] -= coord_x
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        super().__init__(name, weight, coords)
```

Comment:
```suggestion
        super().__init__(name, weight)
```

Remove coords from super, because the parent's class has a default value for it.